po/zh-CN.po

```diff
@@ -6527,24 +6617,6 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
-"```rust,editable\n"
```

Comment:
Curious why the following removed? But I saw some code snippets have added in above changes.

Reply:
It might be a mistake. I will check them later.